File: semester I/Praktikum Pemrograman I/Modul4_JovanGilbertNatamasindah_2310817310002/Python/PRAK403-2310817310002-JovanGilbertNatamasindah.py
ANGKA1, ANGKA2 =  map(int, input("Masukkan dua buah angka : ").split())

i = ANGKA1
j = ANGKA2

if ANGKA1 > ANGKA2:
    for i in range(i, j, -1):
        print(f"{i} {j} -", end=" ")
        j += 1
else:
    for i in range(i, j):
        print(f"{i} {j} -", end=" ")
        j -= 1

print(ANGKA2,ANGKA1)

File: semester I/Praktikum Pemrograman I/Modul4_JovanGilbertNatamasindah_2310817310002/C/PRAK403-2310817310002-JovanGilbertNatamasindah.c
#include <stdio.h>

void main(){
    int ANGKA1, ANGKA2;
    
    printf("Masukkan dua angka dengan spasi : ");
    scanf("%d %d",&ANGKA1, &ANGKA2);

    if (ANGKA1 > ANGKA2) {

        for (int i = ANGKA1, j = ANGKA2; i > ANGKA2; i--,j++) {
            printf("%d %d - ",i, j);
        }
    } 

    else {
        
        for (int i = ANGKA1, j = ANGKA2; i < ANGKA2; i++, j--) {
            printf("%d %d - ",i, j);
        }
    }

    printf("%d %d",ANGKA2, ANGKA1);
}









